README.md

```diff
@@ -1,42 +1,41 @@
 # BunnyHopMaster
-A 3D platformer based on the Source engine movement mechanics, portals, and a basis in speedrunning
+JumpingJax is a 3d platformer video game make using the Unity game engine.
 
-Unity Version: 2019.3.0f3
+The core idea of the game is that the player has tools such as portals and anti-friction boots to find their lost dog.
 
-![Preview](https://github.com/ambid17/BunnyHopMaster/blob/development/MarketingAssets/giphy.gif)
+![Game Preview](https://github.com/ambid17/BunnyHopMaster/blob/development/MarketingAssets/jumpingjax.gif)
```

Comment:
Nothing here
![Game Preview](https://github.com/ambid17/BunnyHopMaster/blob/development/MarketingAssets/jumpingjax.gif)

Reply:
updated

Reply:
Doesn’t seem to work on mobile